### DISCUSSION TITLE
need bare minimum HW baseline based on bandwidth/throughput, especially on memory

### DISCUSSION THREAD

**Discussion by ganduulgag** [Security-Onion-Solutions/securityonion#6218] [Unsupported Versions]
Hi all,

As we run Distributed deployment, we have to decide on the bare minimum hardware baseline on the Forward and the Search nodes based on bandwidth/throughput. Could you please share your hardware performance here with the community so we can draw some hardware scaling conclusions from it, especially for the memory on the Forward and the Search nodes? Thanks in advance.

kind regards,
Duulga

**Comment by dougburks** [comment]
https://docs.securityonion.net/en/2.3/hardware.html

**Reply by ganduulgag** [reply]
thank @dougburks. what I meant here is that let's collect actual performance data based on throughput/bandwidth which will be helpful for coming up with hardware requirements for everyone.

**Reply by isaacgolding** [reply]
I think this is what you are looking for.  
Machine Specs:  Proxmox VM,  32 Cores, 64GB Memory , 2TB (SAS 10k raid 5)  4 Monitor NICS
Network load:   115+- devices on network+ internet traffic.
Monitor traffic:   Max 332.0 Mb/s	Mean 10.7 Mb/s	Last 10.2 Mb/s   (Last 7 days) (as reported from Grafana)
Memory in use average 80% 
NSM looses 5% on average of its space per day.  NSM has 1.7 TB total space.
What other specs are you looking for?
